### PR TITLE
#3950 seed recovery refactor

### PR DESCRIPTION
**Pull request by AlexeyTuboltsev**

### CODE REVIEW

**Comment by andywalton** [comment]
Hi @AlexeyTuboltsev

I have a small changes please:

Screenshot 2020-02-19 14 18 50

1. Add top and bottom left border-radius of 4px for the `.SeedRecovery-unsafe-module__word-number`
2. Change the label of the `Save & close` button to Cancel. Also, please adjust the button icon height and width from 1.2em to 24px. The X icon is being cut off weirdly.
3. When the disabled `Verify` button is enabled, it displays an outline button. Change this to a primary button as per the below.

Screenshot 2020-02-19 14 26 35

4. On step 2, as per point 3 above, use a primary button instead of an outline button.

5. The success modal when the user lands in app, the modal content should change:

From:
Screenshot 2020-02-19 14 28 55

To this layout:
Screenshot 2020-02-19 14 32 47

1. The success tick icon (see attached SVG)
2. Headline and body copy
3. Primary call to action button

Thanks

File: packages/web/app/components/shared/forms/fields/VirtualizedSelect.modules.scss [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  align-items: center;
+  padding-left: 0!important;
+
+  &:focus {
+    outline: none;
+  }
+}
+
+.Select-value-label {
+  font-family: Roboto, sans-serif;
```
Comment (Moejoe90) [inline packages/web/app/components/shared/forms/fields/VirtualizedSelect.modules.scss]: can't we use our typography?

File: packages/web/app/components/shared/forms/fields/VirtualizedSelect.modules.scss [hunk 2]
```diff
@@ -0,0 +1,86 @@
+// sass-lint:disable-all
```
[the remark this answers is not in the document]
Comment (AlexeyTuboltsev) [reply packages/web/app/components/shared/forms/fields/VirtualizedSelect.modules.scss]: comment added

File: packages/web/app/components/shared/forms/fields/VirtualizedSelect.modules.scss (hunk 1 above)
Comment (AlexeyTuboltsev) [reply packages/web/app/components/shared/forms/fields/VirtualizedSelect.modules.scss]: I'm only overriding the font and the color here, everything else should stay the same

File: packages/web/app/components/wallet-selector/light/Login/MissingEmailLightWallet.tsx [hunk 3]
```diff
@@ -16,7 +16,7 @@ export const MissingEmailLightWallet: React.FunctionComponent<{}> = () => (
       <FormattedHTMLMessage
         tagName="span"
         id="wallet-selector.neuwallet.login-instructions"
-        values={{ url: recoverRoutes.help }}
+        values={{ url: appRoutes.restore }}
```
[the remark this answers is not in the document]
Comment (AlexeyTuboltsev) [reply packages/web/app/components/wallet-selector/light/Login/MissingEmailLightWallet.tsx]: yes, according to the new design and flow

File: packages/web/app/modules/wallet-selector/light-wizard/sagas.ts [hunk 4]
```diff
@@ -184,9 +186,14 @@ export function* lightWalletRecoverWatch(
       password,
     );
     yield web3Manager.plugPersonalWallet(wallet);
-    yield neuCall(logoutUser);
```
[the remark this answers is not in the document]
Comment (AlexeyTuboltsev) [reply packages/web/app/modules/wallet-selector/light-wizard/sagas.ts]: I changed it according to the new flow

**Pull request merged**